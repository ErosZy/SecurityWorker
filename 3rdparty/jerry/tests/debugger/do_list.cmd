b do_list.js:18
b do_list.js:19
list
quit
