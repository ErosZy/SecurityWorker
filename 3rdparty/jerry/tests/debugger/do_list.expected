Connecting to: localhost:5001
Stopped at tests/debugger/do_list.js:15
(jerry-debugger) b do_list.js:18
Breakpoint 1 at tests/debugger/do_list.js:18
(jerry-debugger) b do_list.js:19
Breakpoint 2 at tests/debugger/do_list.js:19
(jerry-debugger) list
=== Active breakpoints  ===
 1: tests/debugger/do_list.js:18
 2: tests/debugger/do_list.js:19
(jerry-debugger) quit
